server/utils/nunjucksSetup.ts

```diff
@@ -77,16 +77,16 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     return null
   })
 
-  njkEnv.addFilter('toSelect', (array, valueKey, textKey, value) => {
+  njkEnv.addFilter('toSelect', (array, valueKey, textKey, value, emptyOptionText) => {
```

Comment:
We now allow string values - not just numbers or string representation of numbers